tests/Feature/app/Http/Controllers/OrderRouteTest.php

```
@@ -28,7 +28,7 @@ public function it_shows_order_details_to_authenticated_user(): void
 
         $order = Order::factory()->createQuietly(['customer_id' => $user->id]);
 
-        $response = $this->get(route('orders.show', $order));
+        $response = $this->get(route('web.orders.show', [$order->uuid]));
         $response->assertOk();
         $response->assertSee($order->title);
     }
```

PHPMD (2.15.0)

The method it_shows_order_details_to_authenticated_user is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)